qiskit/quantum_info/states/statevector.py

```diff
@@ -249,6 +249,20 @@ def is_valid(self, atol: float | None = None, rtol: float | None = None) -> bool
         norm = np.linalg.norm(self.data)
         return np.allclose(norm, 1, rtol=rtol, atol=atol)
 
+    def is_entangled(self, epsilon: float = 1e-10) -> bool:
```

Comment:
it is better to use the existing atol/rtol parameters than epsilon (see the `is_valid` function above)